Lib/glyphsLib/builder/axes.py

```diff
@@ -520,8 +520,18 @@ def get_regular_master(font):
     """
     if not font.masters:
         return None
-    regular_name = font.customParameters["Variation Font Origin"]
-    if regular_name is not None:
+    # The current glyphs source specification supports the custom
+    # parameter name "Variable Font Origin".  This may have been
+    # named "Variation Font Origin" in the past.
+    # We support the current name with a fallback to the previous name
+    # if not found in the GSFont.customParameters dict
+    if "Variable Font Origin" in font.customParameters:
+        regular_id = font.customParameters["Variable Font Origin"]
+        for master in font.masters:
+            if master.id == regular_id:
+                return master
+    elif "Variation Font Origin" in font.customParameters:
+        regular_name = font.customParameters["Variation Font Origin"]
```

Comment:
I would check that the result from customParameters is not None (it's None when a requested parameter is missing), just in case the the attribute you're comparing the result with is also None and you get a false positive (not sure if master.name or master.id can ever be None, but better be safe).
Also you can issue the logging warning in here, after you found the old Variation Font Origin

Reply:
84077755fb867de058decbf5a06965ba132a9caf